util/delete-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Let's update [this README](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/deploy#reindex-with-no-downtime).
```
kubernetes-elasticsearch-cluster/create-cluster.sh DATASET
kubernetes-elasticsearch-cluster/deploy-es.sh DATASET
# Verily elasticsearch was deployed
kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
bigquery/deploy/deploy-indexer.sh DATASET
# Verily indexing was successful
kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
# Run from data-explorer repo
deploy/deploy-api.sh DATASET
# If everything looks good, delete old cluster. Run from this repo:
util/delete-cluster.sh DATASET
```
And delete the "After verifying Data Explorer UI works..."  line.